en/move.json

```
@@ -529,51 +529,51 @@
   },
   "amnesia": {
     "name": "Amnesia",
-    "effect": "The user temporarily empties its mind to forget its concerns. This sharply raises the user’s Sp. Def stat."
+    "effect": "The user temporarily empties its mind to forget its concerns. This sharply boosts the user’s Sp. Def stat."
   },
   "kinesis": {
     "name": "Kinesis",
     "effect": "The user distracts the target by bending a spoon. This lowers the target’s accuracy."
   },
   "softBoiled": {
     "name": "Soft-Boiled",
-    "effect": "The user restores its own HP by up to half of its max HP."
+    "effect": "The user restores its own HP by up to half its max HP."
   },
   "highJumpKick": {
     "name": "High Jump Kick",
-    "effect": "The target is attacked with a knee kick from a jump. If it misses, the user is hurt instead."
+    "effect": "The target is attacked with a knee kick from a jump. If this move misses, the user takes damage instead."
   },
   "glare": {
     "name": "Glare",
-    "effect": "The user intimidates the target with the pattern on its belly to cause paralysis."
+    "effect": "The user intimidates the target with terrifying eyes to cause paralysis."
```

```suggestion
    "effect": "The target is transfixed with terrifying sharp eyes. The target is frightened into paralysis."
```
I think Pokemon Stadium's description sounds really cool, accomplishes the general theme and is an official description for the few that care. Though the current change is serviceable.